test/test_overrides.py

```diff
@@ -0,0 +1,814 @@
+    (torch.sqrt, lambda input, out=None: -1),
+    (torch.squeeze, lambda input, dim=None, out=None: -1),
+    (torch.sspaddmm, lambda input, mat1, mat2, beta, alpha, out=None: -1),
+    (torch.stack, lambda tensors, dim=0, out=None: -1),
+    (torch.std, lambda input: -1),
+    (torch.std_mean, lambda input: -1),
+    (torch.stft, lambda input, n_fft, hop_length=None, win_length=None, window=None, center=True, pad_mode='reflect',
+     normalized=False, onesided=True: -1),
+    (torch.sub, lambda input, other, out=None: -1),
+    (torch.sum, lambda input: -1),
+    (torch.svd, lambda input, some=True, compute_uv=True, out=None: -1),
+    (torch.symeig, lambda input, eigenvectors=False, upper=True, out=None: -1),
+    (torch.t, lambda input: -1),
+    (torch.take, lambda input, index: -1),
+    (torch.tan, lambda input, out=None: -1),
+    (torch.tanh, lambda input, out=None: -1),
+    (torch.tensordot, lambda a, b, dims=2: -1),
+    (torch.threshold, lambda input, threshold, value, inplace=False: -1),
+    (torch.topk, lambda input, k, dim=-1, descending=False, out=None: -1),
+    (torch.trace, lambda input: -1),
+    (torch.transpose, lambda input, dim0, dim1: -1),
+    (torch.trapz, lambda y, x, dim=-1: -1),
+    (torch.triangular_solve, lambda input, A, upper=True, transpose=False, unitriangular=False: -1),
+    (torch.tril, lambda input, diagonal=0, out=None: -1),
+    (torch.tril_indices, lambda row, col, offset=0, dtype=torch.long, device='cpu', layout=torch.strided: -1),
+    (torch.triplet_margin_loss, lambda anchor, positive, negative, margin=1.0, p=2, eps=1e-06, swap=False, size_average=None,
+     reduce=None, reduction='mean': -1),
+    (torch.triu, lambda input, diagonal=0, out=None: -1),
+    (torch.triu_indices, lambda row, col, offset=0, dtype=torch.long, device='cpu', layout=torch.strided: -1),
+    (torch.trunc, lambda input, out=None: -1),
+    (torch.unbind, lambda input, dim=0: -1),
+    (torch.unique, lambda input, sorted=True, return_inverse=False, return_counts=False, dim=None: -1),
+    (torch.unique_consecutive, lambda input, return_inverse=False, return_counts=False, dim=None: -1),
+    (torch.unsqueeze, lambda input, dim, out=None: -1),
+    (torch.var, lambda input: -1),
+    (torch.var_mean, lambda input: -1),
+    (torch.where, lambda condition, x, y: -1),
+    (torch.zeros_like, lambda input, dtype=None, layout=None, device=None, requires_grad=False: -1),
+)
+
+TENSOR_LIKE_OVERRIDES = tuple(t[0] for t in TENSOR_LIKE_TORCH_IMPLEMENTATIONS)
+
+# TODO make these functions overridable
+TENSOR_LIKE_SKIP_TESTS = (
+    torch.sparse_coo_tensor,
+)
+
+def generate_tensor_like_torch_implementations():
+    torch_vars = vars(torch)
+    untested_funcs = []
+    for func_name in torch.__all__ + dir(torch._C._VariableFunctions):
+        # ignore private functions or functions that are deleted in torch.__init__
+        if func_name.startswith('_') or func_name == 'unique_dim':
```

Comment:
What's going on with `unique_dim`?

Reply:
`unique_dim` gets deleted explicitly (https://github.com/pytorch/pytorch/blob/master/torch/functional.py#L423) so it's not actually in the torch namespace. It is a low-level kernel though so it is in `dir(torch._C._VariableFunctions)`. I could have made the loop over `vars(torch)` but then I'd need to exclude a bunch of other stuff (e.g. dunder methods, magic methods) that it seemed simpler to just have a special case for `unique_dim`.